drivers/net/ethernet/intel/idpf/idpf_virtchnl.h

```diff
@@ -8,63 +8,105 @@ struct idpf_adapter;
 struct idpf_netdev_priv;
 struct idpf_vec_regs;
 struct idpf_vport;
-struct idpf_vport_max_q;
+struct idpf_max_q;
```

Comment:
This file content is no longer reflecting the virtchnl code. So you may move all this code into eth_common.h and get rid of this file.

Also above forward declaration are not required as they are not used in this file.

Reply:
Moved to idpf_eth_common.h.